vividus/src/main/java/org/vividus/transformer/AbstractFilteringTableTransformer.java

```diff
@@ -30,10 +32,17 @@
 
     protected List<String> filterColumnNames(List<String> allColumnNames, String byColumnNames)
     {
-        List<String> filtered = new ArrayList<>(allColumnNames);
-        filtered.retainAll(Stream.of(StringUtils.split(byColumnNames, ';'))
+        List<String> selectedColumnNames = Stream.of(StringUtils.split(byColumnNames, ';'))
                 .map(String::trim)
-                .collect(Collectors.toList()));
+                .collect(Collectors.toList());
+
+        String undefinedColumns = selectedColumnNames
+                .stream().filter(element -> !allColumnNames.contains(element)).collect(Collectors.joining(" "));
+        isTrue(undefinedColumns.isEmpty(),
+                "'byColumnNames' refers columns missing in ExamplesTable: %s", undefinedColumns);
```

Comment:
I think it makes sense to use line separator as separator for column names because we may want to use space as part of column name e.g.
'byColumnNames' refers columns missing in ExamplesTable:
col1
col2

Reply:
discussed to use ; instead of space as it used less often

Reply:
> discussed to use ; and space instead of space as it used less often

Yes, I think ';' + 'space symbol' will be more correct and readable